src/app/shared/iframe/iframe.ts

```diff
@@ -3,6 +3,10 @@ export enum BubbleToIonicPostMessage {
   IFRAME_BACK_BUTTON_CLICKED = 'iframe-back-button-clicked',
   EDIT_CAPTION_CANCEL = 'edit-caption-cancel',
   EDIT_CAPTION_SAVE = 'edit-caption-save',
+  IFRAME_BUY_NUM_BUTTON_CLICKED = 'iframe-buy-num-button-clicked',
+  IFRAME_COPY_TO_CLIPBOARD_ASSET_WALLET = 'iframe-copy-to-clipboard-asset-wallet',
+  IFRAME_COPY_TO_CLIPBOARD_INTEGRITY_WALLET = 'iframe-copy-to-clipboard-integrity-wallet',
+  IFRAME_COPY_TO_CLIPBOARD_PRIVATE_KEY = 'iframe-copy-to-clipboard-private-key',
 }
```

Comment:
**by ChatGPT** 
The code seems to be defining an enum for different post messages being sent from an iframe. There is nothing inherently wrong with this code, but some possible improvements could be:

1.   Adding comments to explain what each enum value means and when it should be used.
1.   Using more descriptive enum values instead of abbreviations to make it easier to understand the purpose of each value.
1.   Consolidating any duplicate enum values into a single value to reduce redundancy and improve clarity.

Reply:
@tammyyang regarding ChatGPT suggestions:
1. to me it's completely selfexplanatory however for other devs or form me (after 2-3 month) it might make sense to have comments. 
2. opinionated.
3. I see no duplicates.

Reply:
Agreed, they are selfexplanatory but a task to improve is even better